code/modules/awaymissions/maploader/reader.dm

```diff
@@ -220,9 +220,12 @@ var/global/dmm_suite/preloader/_preloader = new
 	var/atom/instance
 	_preloader.setup(attributes, path)
 
-	var/turf/T = locate(x,y,z)
-	if(T)
-		instance = new path (T)//first preloader pass
+	var/turf/oldTurf = locate(x,y,z)
+	if(ispath(path, /turf)) //Turfs use ChangeTurf
+		if(path != oldTurf.type)
+			instance = oldTurf.ChangeTurf(path)
```

Comment:
This is going to lag the shit out of the server. Recalculating lighting and LINDA for every single turf in a map.

So, why? What's the problem with initializing lighting after the fact, like it does [here](https://github.com/ParadiseSS13/Paradise/blob/master/code/modules/awaymissions/zlevel.dm#L9)